packages/environment-ember-template-imports/-private/environment/index.ts

```diff
@@ -13,12 +22,15 @@ export default function emberTemplateImportsEnvironment(): GlintEnvironmentConfi
               if: 'if',
               unless: 'if-not',
               yield: 'yield',
+              ...additionalGlobalSpecialForms,
             },
             imports: {
               '@ember/helper': {
                 array: 'array-literal',
                 hash: 'object-literal',
+                ...(additionalSpecialForms.imports?.['@ember/helper'] ?? {}),
               },
+              ...(additionalSpecialForms.imports ?? {}),
```

Comment:
Spreading `null` or `undefined` is just a no-op, so anywhere in these you have `...(something ?? {})` can just be `...something` instead